### PR TITLE
(PE-18318) Update docs for command_broadcast flag

### PR DESCRIPTION
**Pull request by mullr**
This is being set to true by PE.

### CODE REVIEW

**Comment by mullr** [comment]
@melamos here's the puppetdb docs change. look ok?

File: documentation/puppetdb_connection.markdown [hunk 1]
```diff
@@ -101,7 +101,7 @@ The default value is false.
 
 When using multiple `server_urls`, this flag can be set to `true` to cause commands to be sent to all configured PuppetDB instances.
 
-The default value is false.
+The default value is `false` when not specified, but Puppet Enterprise 2016.5 and later explicitly configures `command_broadcast` to `true`.
```
Comment (wkalt) [inline documentation/puppetdb_connection.markdown]: Would it be clearer to say something like "Defaults to 'false' in FOSS and 'true' in Puppet Enterprise > 2016.5". To me the current wording implies that what PE is doing is something other than defaulting, which is a little confusing.

Comment (melamos) [inline documentation/puppetdb_connection.markdown]: How about:

When set to `true` in installations using multiple `server_urls`, commands are sent to all configured PuppetDB instances. 

In open source Puppet and PE versions earlier than 2016.5, the default setting is `false`. In PE 2016.5 and later, the default setting is `true`.

**Comment by wkalt** [comment]
This should ultimately go against stable, but we might want to hold off on merging until the 2016.5 release due to the reference.

**Comment by mullr** [comment]
ok @melamos , updated to use that wording. Any thought about when to push this? doesn't seem like a huge deal for this to go live a little early, but I could be wrong.

**Comment by melamos** [comment]
I don't think we should release information about code changes before the actual product release. Can this PR hang out for a bit and be merged when Everett releases?

**Comment by mullr** [comment]
@melamos sure, that sounds good

**Comment by senior** [comment]
Marking this as "Don't Merge Yet", assuming we still want to wait for mid-December

**Comment by senior** [comment]
@mullr when can this be merged?

**Pull request merged**